pkg/trace/operation.go

```diff
@@ -226,7 +226,11 @@ func opID(opNum uint64) string {
 
 // NewOperation will return a new operation with operationID added as a value to the context
 func NewOperation(ctx context.Context, format string, args ...interface{}) Operation {
-	return newOperation(ctx, opID(atomic.AddUint64(&opCount, 1)), 3, fmt.Sprintf(format, args...))
+	o := newOperation(ctx, opID(atomic.AddUint64(&opCount, 1)), 3, fmt.Sprintf(format, args...))
+
+	frame := o.t[0]
+	o.Debugf("[NewOperation] %s [%s:%d]", o.header(), frame.funcName, frame.lineNo)
```

Comment:
Are you intentionally adding this back in? I seem to recall it was removed during the big operation trace PR.

Reply:
Yes...this is intentional.  If you recall when we were looking into the `fixVM` issue I lamented that I removed it -- so I'm returning it.